### ISSUE TITLE
ImportError: cannot import name 'ReverseSingleRelatedObjectDescriptor'

### ISSUE THREAD

**Issue by Emnalyeriar** [edoburu/django-parler#112]
Hey, 
I get this error after instaling and importing from django-parler. Using python 3.4.4 and django 1.9. Is it not compatible with these versions?

**Comment by Emnalyeriar** [comment]
Ok, just found in Django 1.9 release notes that django.db.models.fields.related classes were renamed. I am using parler 1.5.1 (this is the default version from pip) and It looks like it was fixed in 1.6?

**Comment by vdboor** [comment]
Yes, this issue is fixed in django-parler 1.6!

**Comment by Emnalyeriar** [comment]
I'll hook on to this issiue and ask one more question.
Not sure if I made a mistake somewhere or python, django and parler versions as mentioned above are not compatible.
I made a new app for homepage and added a Configuration model. The model has a OneToOneField with django.contrib.sites.model (so that it can have only one homepage configuration per site) and next fields are translations like this:
```python
class Configuration(TranslatableModel):
    site = models.OneToOneField(Site)
    translations = TranslatedFields(
        header=models.CharField(_('header'), max_length=200)
    )
```
I have all the necessary settings set up. I registered the configuration model for admin and the problem is that I don't see any fields in admin besides the site select.
Did I code something wrong? Or did I omit some settings maybe?
